### ISSUE TITLE
pip install calysto_processing not working

### ISSUE THREAD

**Issue by Guest007** [Calysto/calysto_processing#1]
```
$ pip install calysto_processing
Collecting calysto-processing
  Using cached calysto_processing-2.0.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-build-fHRl7Y/calysto-processing/setup.py", line 49, in <module>
        with open('README.md') as f:
    IOError: [Errno 2] No such file or directory: 'README.md'
```

**Comment by dsblank** [comment]
Thank you for the note; should be fixed with b834f1d58440c5c454371584c7341b3ef52749ee

**Comment by dsblank** [comment]
Updated pypi with calysto_processing 2.0.2.
